### PR TITLE
Multitarget for Microsoft.Extensions.ObjectPool

### PR DESCRIPTION
**Pull request by sungam3r**
See https://github.com/ArieGato/serilog-sinks-rabbitmq/pull/193

### CODE REVIEW

File: src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj [hunk 1]
```diff
@@ -22,6 +21,22 @@
...
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="6.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="7.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="8.0.0" />
```
Comment (ArieGato) [inline src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj:37]: Is it on purpose that you don't use 8.0.3?

Comment (sungam3r) [reply src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj:37]: Yes. I work in different repos. Some bump dependencies frequently, even patches. Some tend to reference only minimum requered for ages. I would not say there is the best strategy for any case.

Comment (ArieGato) [reply src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj:37]: I think that updating the package with dependabot will ensure our package runs on the newest patches. That's the only argument I can think of.

Comment (sungam3r) [reply src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj:37]: So? 8.0.0 or 8.0.3 ? We can merge with 8.0.0 , publish v7 release and then bump versions for v7.0.1.

Comment (ArieGato) [reply src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj:37]: Sure

**Review by ArieGato — APPROVED** [review]

**Pull request merged**